pkg/webhook/server.go

```diff
@@ -369,34 +373,81 @@ func (wh *webhook) handleWebhookValidate(w http.ResponseWriter, req *http.Reques
 	)
 }
 
-func reviewResponse(uid types.UID, err error, aletmanagerHost string, resource string, name string, namespace string) *admissionv1.AdmissionReview {
-	// Currently hard coded and commented logic in order to just audit, but in order to get the content we need the policy to be deny.
-	// allowed := err == nil
-	//var status int32 = http.StatusAccepted
-	// if err != nil {
-	// 	status = http.StatusForbidden
-	// }
+func getValidationAnnotations(attrs admission.Attributes) (audit bool, deny bool) {
+	validationActionsPattern := `validationActions":\[(.*?)\]`
+	regex, _ := regexp.Compile(validationActionsPattern)
+
+	match := regex.FindStringSubmatch(fmt.Sprintf("%+v", attrs))
+	if len(match) >= 2 {
+		actions := match[1]
+		audit = strings.Contains(actions, "Audit")
+		deny = strings.Contains(actions, "Deny")
+	}
+
+	logger.Info("The actions are", "audit", audit, "deny", deny)
+
+	return audit, deny
+}
```

Comment:
What are the default values of `audit` and `deny`?

Reply:
false